internal/configs/version1/nginx-plus.tmpl

```diff
@@ -26,9 +26,7 @@ load_module modules/ngx_http_app_protect_dos_module.so;
 {{$value}}{{end}}
 {{- end}}
 
-{{if .PreviewPolicies}}
```

Comment:
This was a "workaround" to avoid loading the module all the time when it's not actually needed (only OIDC requires it). I'm not sure if want to always load it now and how else we can easily control it.